### PR TITLE
Changing gtest to CamelCase

### PR DESCRIPTION
**Pull request by avnish98**
I have changed all gtest names from _ convention (like wallet_zkeys_test) to CamelCase Convention ( like WalletZkeysTest) in directories src/gtest and src/wallet/gtest/

### CODE REVIEW

File: src/gtest/test_libzcash_utils.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include <gtest/gtest.h>
 #include "zcash/util.h"
 
-TEST(libzcash_utils, convertBytesVectorToVector)
+TEST(LibzcashUtils, onvertBytesVectorToVector)
```
Comment (str4d) [inline src/gtest/test_libzcash_utils.cpp]: Fix typo: `onvertBytesVectorToVector -> ConvertBytesVectorToVector`

File: src/gtest/test_tautology.cpp [hunk 2]
```diff
@@ -1,10 +1,10 @@
 #include <gtest/gtest.h>
 
-TEST(tautologies, seven_eq_seven) {
+TEST(tautologies, SevenEqSeven) {
     ASSERT_EQ(7, 7);
 }
 
-TEST(tautologies, DISABLED_ObviousFailure)
+TEST(tautologies, DISABLEDObviousFailure)
```
Comment (str4d) [inline src/gtest/test_tautology.cpp]: Please undo this change. `DISABLED_` is the [documented syntax for disabling a test in GoogleTest](https://github.com/google/googletest/blob/master/googletest/docs/faq.md#how-do-i-temporarily-disable-a-test).

**Comment by Eirik0** [comment]
Hey @avnish98 thanks for doing this! Sorry this got lost in a see of other priorities. Are you still available to fix the merge conflicts? It would be absolutely wonderful to get this inconsistency resolved. I've moved this up on the priority list.

Thank you so much again for taking the time to work on this!

**Comment by avnish98** [comment]
@Eirik0 Yes I am still available but I would need at least 5 days to resolve issues, I am involved with some other projects as well. I hope you won't mind.

**Comment by Eirik0** [comment]
@avnish98 Don't mind at all. Take you're time. :)
